app/controllers/items_controller.rb

```diff
@@ -1,4 +1,27 @@
 class ItemsController < ApplicationController
+  before_action :authenticate_user!, only: [:new]
+
   def index
+    @items = Item.all
+  end
+
+  def new
+    @item = Item.new
+  end
+
+  def create
+    @item = Item.create(item_params)
```

Comment:
```suggestion
    @item = Item.new(item_params)
```
`createメソッド = new + save`ですので、上記のように変更しましょう。